### PR TITLE
E2E Form management workflow

### PR DESCRIPTION
**Pull request by sherrif10**

### CODE REVIEW

**Comment by sherrif10** [comment]
cc @kdaud

File: src/test/java/org/openmrs/kenyaemr/contrib/qaframework/automation/page/ClinicianHomePage.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    private static final By FIRST_PATIENT = By.xpath("/html/body/div[3]/div[2]/div/div[2]/div[1]/table/tbody/tr/td[2]");
+
+    public ClinicianHomePage(Page parent) {
+        super(parent);
+    }
+
+    public clinicalViewPatientPage clickOnFirstPatient(){
+        clickOn(FIRST_PATIENT);
+        return new clinicalViewPatientPage(this);
+    }
```
Comment (kdaud) [inline src/test/java/org/openmrs/kenyaemr/contrib/qaframework/automation/page/ClinicianHomePage.java:18]: adding  an empty line could probably be a good ideal.

File: src/test/java/org/openmrs/kenyaemr/contrib/qaframework/automation/page/EnterFormsPage.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        clickOn(ENTER_FORM);
+        return new clinicalViewPatientPage(this);
+    }
+
+    public void FillInAntenalForm(String number, String Appointment) {
+          setTextToFieldNoEnter(REGISTER_NUMBER, number);
+          setTextToFieldNoEnter(APPOINTMENT_DATES, Appointment);
+    }
+    
+
```
Comment (kdaud) [inline src/test/java/org/openmrs/kenyaemr/contrib/qaframework/automation/page/EnterFormsPage.java:40]:
```suggestion
```

**Comment by kdaud** [comment]
Great work @sherrif10 

Have skimmed through the changes and left minor comment to address otherwise its worth merging.

**Comment by sherrif10** [comment]
Thanks @kdaud  for the review.

**Pull request merged**